### DISCUSSION TITLE
Score with subwoofer and EQ

### DISCUSSION THREAD

**Discussion by duchieu235** [pierreaubert/spinorama#283] [Q&A] [answered]
I have read the help page but I still don't really understand how the "with subwoofer and EQ" score is calculated
![image](https://github.com/user-attachments/assets/d609a793-373b-48c8-8160-9893761c1d56)

**Comment by pierreaubert** [comment, marked as answer]
The score has 4 components and depends heavily of the behaviour of the speaker in the bass area. The lower your speaker can go the higher the score. It make it difficult to compare a bookshelves and a floorstander. The idea of the score with subwoofer is to be able to compare the 2 if you add a subwoofer to the bookshelves. That's also useful for HT optimised speakers where they want high SPL and dont want to go low because of that. They all use subwoofers and design the speaker taking into account the sub. Technically the core is modified as if you added a perfect subwoofer to your bookshelves. 

Is that more clear? I can dig out a long thread on ASR about the topic.
